client/app/telecom/pack/xdsl/modem/templates/config/pack-xdsl-modem-templates-config.html

```diff
@@ -0,0 +1,250 @@
+<div class="telecom-pack-xdsl-wifi-config">
+
+    <div class="text-center" data-ng-if="$ctrl.loaders.template">
+        <oui-spinner></oui-spinner>
+    </div>
+
+    <div data-ng-if="!$ctrl.loaders.template">
+
+        <!-- Error Display -->
+        <div data-ng-if="$ctrl.errors.template"
+                class="alert alert-warning"
+                role="alert"
+                data-translate="xdsl_modem_template_config_error_missing">
+        </div> <!-- End of Error Display -->
+
+        <!-- Templates list display -->
+        <div class="row col-xs-12"
```

Comment:
You cannot use both classes `row` and `col-*-*` on the same element.

Furthermore using a row and a col-*-12 is like using a single `div` (block) element.

```suggestion
        <div
```